contactsummary/js/src/contact-summary/services/leave.js

```diff
@@ -261,6 +261,7 @@ define([
                     return period;
                 });
         }
+        factory.getCurrentPeriod = getCurrentPeriod;
```

Comment:
`getCurrentPeriod` is not a private method anymore since you are referencing it in the `factory` object, so just move the implementation directly in `factory.getCurrentPeriod` and reference it in the private methods